tools/fetch_elfutils.sh

```diff
@@ -37,8 +37,7 @@ TAR_ELF="elfutils-${VER_ELF}.tar.bz2"
 URL_ELF="https://sourceware.org/elfutils/ftp/${VER_ELF}/${TAR_ELF}"
 
 if [ -e ${TARGET_EXTRACT} ]; then
-    echo "Error, clean the directory : ${TARGET_EXTRACT}"
-    exit 1
+    rm -rf ${TARGET_EXTRACT}
```

Comment:
Am I understanding this correctly?  Whenever cmake is run and the elfutils directory already exists, then:
1. Delete the directory
2. Download the tarball again
3. build the efultils dependency from scratch again